api/app/wildfire_one/wfwx_api.py

```diff
@@ -204,10 +244,9 @@ async def get_hourly_actuals_all_stations(
 
     for hourly in hourlies:
         if hourly.get('hourlyMeasurementTypeCode', '').get('id') == 'ACTUAL':
-            parsed_hourly = parse_hourly(hourly)
             try:
                 station_code = station_code_dict[(hourly['stationId'])]
-                hourly_actual = parse_hourly_actual(station_code, parsed_hourly)
+                hourly_actual = parse_hourly_actual(station_code, hourly)
```

Comment:
codecov is saying there's no test coverage at this line - seems like it's something we should be testing?

Reply:
It's existing code without tests but I can add some.